docs_src/training.ipynb

```diff
@@ -27,29 +27,29 @@
     "hide_input": false
    },
    "source": [
-    "The fastai library is structured training around a [`Learner`](/basic_train.html#Learner) object that binds together a pytorch model, some data with an optimizer and a loss function, which then will allow us to launch training.\n",
+    "The fastai library structures its training process around the [`Learner`](/basic_train.html#Learner) class, whose object binds together a PyTorch model, a dataset, an optimizer, and a loss function; the entire learner object then will allow us to launch training.\n",
     "\n",
-    "[`basic_train`](/basic_train.html#basic_train) contains the definition of this [`Learner`](/basic_train.html#Learner) class along with the wrapper around pytorch optimizer that the library uses. It defines the basic training loop that is used each time you call the [`fit`](/basic_train.html#fit) function in fastai (or one of its variants). This training loop is kept to the minimum number of instructions, and most of its customization happens in [`Callback`](/callback.html#Callback) objects.\n",
+    "[`basic_train`](/basic_train.html#basic_train) defines this [`Learner`](/basic_train.html#Learner) class, along with the wrapper around the PyTorch optimizer that the library uses. It documents the basic training loop that is used each time you call the [`fit`](/basic_train.html#fit) function(or one of its variants) in fastai. A training loop method requires an intentionally minimal number of arguments; if you want to customize your training loop, you could include in the arguments an optional [`Callback`](/callback.html#Callback) object, which provides a rich number of customization options for the training loop.\n",
```

Comment:
I would leave "It defines the basic training loop" (not documents). The last sentence really needs to be "This training loop is kept to the minimum number of instructions, and most of its customization happens in [`Callback`](/callback.html#Callback) objects." "A training loop method requires an intentionally minimal number of arguments" isn't accurate and would be misleading. Then at the end you can add "you can include such Callbacks in the arguments as an optional parameter"

Reply:
Thanks for the detailed reply. I get the last part but could you please elaborate a bit on "This training loop is kept to the minimum number of instructions"? Do you mean that the training loop is very bare bone and generic? Or do you intend to mean that `fit` takes very few arguments to call? Or maybe both? Sorry for the back-and-forth but I really want to make sure that I understand exactly what you mean.

Reply:
Yes, I mean the training loop has very few lines of code. The number of arguments isn't what Callbacks are for, they are there to avoid having thousands of lines of code in the basic training loop.

Reply:
Ah, I see. Thank you again for the explanation!